core/core_test.go

```diff
@@ -50,7 +50,7 @@ func TestCreate(t *testing.T) {
 
 func TestSummary(t *testing.T) {
 	createJson := []byte("{\"Name\":\"Alice\",\"Password\":\"Hello\"}")
-	delegateJson := []byte("{\"Name\":\"Bob\",\"Password\":\"Rob\",\"Time\":\"2h\",\"Uses\":1}")
+	delegateJson := []byte("{\"Name\":\"Bob\",\"Password\":\"Rob\",\"Time\":\"2h\",\"Uses\":1,\"AnyUser\":true}")
```

Comment:
It would be useful to test a lot of these with `AnyUser: false` to ensure that it fails in that case.